packages/verification-vue-component/src/style/components/verification/card/_help.scss

```diff
@@ -0,0 +1,200 @@
+.help-card {
+    .card-header {
+        padding: 7px 4px;
+        font-weight: bold;
+    }
+
+    .card-body {
+        form {
+            padding: 0 4px;
+
+            p.consent-field {
+                display: flex;
+            }
+
+            label {
+                display: inline-block;
+                color: rgba(#14213D, 0.5);
+                font-size: 13px;
+                line-height: 1.5em;
+                font-weight: 600;
+
+                &.consent {
+                    font-size: $font-size-medium;
+                }
+            }
+
+            #email, #question {
```

Comment:
Never use id's in css if you don't really need it, it can cause all sorts of problems with inheritation if you use id's.

Reply:
Adjusted it